src/components/input-style/input-style.scss

```diff
@@ -21,18 +21,11 @@
     border-radius: 3px;
     padding: $m-input-style--padding;
     transition: border-color $m-transition-duration--l ease-in-out;
-    margin-top: 0.6em;
+    transition: margin-top $m-transition-duration ease;
```

Comment:
C'est normal les 2 transitions une après l'autre?

Reply:
Of course not. Good catch ;)

Reply:
En français... SVP!!! 
(Tu rates jamais aucune occasion de parler en anglais, hein? ;) )